lib/converters.js

```diff
@@ -25,6 +25,22 @@ function indent(content, spaces, ignoreFirst) {
     }).join("\n");
 }
 
+function emphasisFormats( text ) {
+  return {
+    strong: '**',
```

Comment:
Sounds good, @IonicaBizau how about adding a `.editorconfig` [file](http://editorconfig.org/) in order to allow others to have your same code style in your repo, is something I use usually to allow other follow same rules in code style. 

Let me know your comments if so to add an `.editorconfig` file.